portfolio/src/main/webapp/script.js

```diff
@@ -12,12 +12,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/*
+* Sets a time of non-doing for a function
+*/
+
+
```

Comment:
This comment seems out of place since there is another comment below. I think you can remove it :)